docs/src/main/sphinx/release/release-362.md

```diff
@@ -0,0 +1,7 @@
+# Release 362 (dd MMM yyyy)
+
+## Hive connector
+
+* Enable S3 insert overwrite when `hive.insert-existing-partitions-behavior=OVERWRITE` is set
```

Comment:
We do not edit release notes directly in the PR. 
When PR is merged one has to add entry to Release notes issue. For 362: https://github.com/trinodb/trino/issues/9015

Reply:
Ok thx. Will revert it back then.